build.gradle

```diff
@@ -1,9 +1,10 @@
 apply plugin: 'java'
 apply plugin: 'eclipse'
+apply plugin: 'idea'
```

Comment:
please do not commit this file. I'll deal with the version changes / api compilation